### PR TITLE
Add support for elliptical ROI

### LINKED ISSUE #31: Support ellipse in region translator
**Is your feature request related to a problem? Please describe it:**

Cannot draw ellipse.

**Describe the solution you'd like:**

Add support for https://astropy-regions.readthedocs.io/en/stable/api/regions.EllipsePixelRegion.html and `EllipticalROI` in `glue-jupyter`.

**Additional context:**

glue-viz/glue-jupyter#223

spacetelescope/jdaviz#635 (Imviz MVP)

cc @javerbukh

### PR DESCRIPTION
**Pull request by javerbukh**
# Pull Request Template

## Description
Potentially fixes #31 

I'm not sure if anything else needs to be done here to implement elliptical ROI in glue-astronomy, and by extension jdaviz. Testing on my machine does not look different but I'm not sure if further changes need to be made in glue-jupyter or jdaviz.

### CODE REVIEW

File: glue_astronomy/translators/regions.py [hunk 1]
```diff
@@ -82,6 +81,8 @@ def to_object(self, subset):
                 return PolygonPixelRegion(PixCoord(roi.vx, roi.vy))
             elif isinstance(roi, CircularROI):
                 return CirclePixelRegion(PixCoord(*roi.get_center()), roi.get_radius())
+            elif isinstance(roi, EllipticalROI):
+                return EllipsePixelRegion(PixCoord(roi.xc, roi.xy), roi.radius_x, roi.radius_y)
```
Comment (rosteen) [inline glue_astronomy/translators/regions.py:85]:
```suggestion
                return EllipsePixelRegion(PixCoord(roi.xc, roi.yc), roi.radius_x, roi.radius_y)
```

Wrong attribute name, works after this change.

**Review by rosteen — CHANGES_REQUESTED** [review]
Needs one minor change, but works after that. I'm not sure what the CI failures are, I see "build not found" when I click to try to see details. @eteq any input here?

File: glue_astronomy/translators/regions.py (hunk 1 above)
Comment (javerbukh) [reply glue_astronomy/translators/regions.py]: Good call! Pushed the fix in the latest commit.

**Comment by pllim** [comment]
I also wonder how this will affect spacetelescope/jdaviz#697 🤔

**Review by eteq — APPROVED** [review]
LGTM!

The failures are due to an issue with the CI setup on linux *not* this PR itself. It's passing on windows though, so I'm pretty confident there's not a concern there.

**Comment by eteq** [comment]
(also that "build not found" error is weird, but will troubleshoot with @rosteen  out-of-band)

**Comment by pllim** [comment]
> It's passing on windows though, so I'm pretty confident there's not a concern there.

See, Windows is useful sometimes. 😆

**Pull request merged**